### PR TITLE
feat(taikoon): ui qa checklist

### PR DESCRIPTION
**Pull request by bearni95**
- [x] Background, light mode:
  - [x] Remove pink gradients
  - [x] Bg color: `#f8f8f8`
- [x] Blur header background once scroll starts
- [x] Apply custom style to navigation menu buttons
- [x] Align nav buttons as the only centered element on the top bar
- [x] "Mint now" button custom styles
- [x] Replicate taiko.xyz hover animation for "Mint now"
- [x] Change "WTF" to "Learn more"
- [x] `Information.section`: Font size corrections
- [x]  Change headline on faq sections, "WTF?" > "FAQs"
  - [x] Style corrections on FAQ items
- [x] Ensure display space for footer on all resolutions
- [x] Mimic animations from taiko.xyz footer
- [x] Vertical alignment for Mint page/section
- [x] Mint page:
  - [x] Simplified design implementation
  - [x] Light mode bg `#f8f8f8`
  - [x] Post-mint modal design implementation
  - [x] Mint progressbar design adjusments
  - [x] mobile: fix spacing between nav menu and bg panel
  - [x] Gas fee calculation error on full refresh
  - [x] Mobile modals
- [x] Sticky scrolling
- [x] Update FAQ content from figma
- [x] Hide menu buttons from countdown landing
- [x] ~Collection preview component for landing~
- [x] Footer link coloring corrections
- [x] Post-mint modal: implement carousel on Taikoon slider view
- [x] Collection page
  - [x] Add title
  - [x] Remove filter and sort
- [x] Mobile
  - [x] Menu copyright section padding adjustment
  - [x] On countdown, replace menu button with theme toggle button

### CODE REVIEW

File: packages/taikoon/data/whitelist/holesky.json
Comment (KorbinianK) [inline packages/taikoon/data/whitelist/holesky.json:1]: Should probably not be public? @bearni95

File: packages/taikoon-ui/src/lib/util/ZeroXAddress.ts
Comment (KorbinianK) [inline packages/taikoon-ui/src/lib/util/ZeroXAddress.ts:1]: import { zeroAddress } from 'viem';

File: packages/taikoon-ui/src/types.ts [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  bidder: string;
+  settled: boolean;
+  //minBid: number
+  bids: IBid[];
+  blockNumber: number;
+  lastUpdate: Date;
+  updateCounter: number;
+}
+
+export type IAddress = `0x${string}`;
```
Comment (KorbinianK) [inline packages/taikoon-ui/src/types.ts:37]: Use this instead:

`import type { Address } from 'viem';`

**Comment by KorbinianK** [comment]
@bearni95 Please add `svelte-i18n`, while we may not initially have other languages, it is a lot easier later on if it was prepared right away. It's also easier to adjust text in a single file rather than dozens.

File: packages/taikoon-ui/src/components/core/Text/H0.svelte [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<script lang="ts">
+  import { classNames } from '$lib/util/classNames';
+
+  import { CommonTextClasses } from './common';
+
+  let classes: string = null;
+  export { classes as class };
+</script>
+
+<div class={classNames(CommonTextClasses, 'font-medium', 'text-h0', classes)}>
```
Comment (KorbinianK) [inline packages/taikoon-ui/src/components/core/Text/H0.svelte:10]: Two things:

- Why not use plain CSS? Just use tailwind @apply rules for your class. This just creates bloat.
- You can access classes via svelte `$$props.class`, no need to export it. See: https://svelte.dev/repl/6de3c939fcbf4c90a0309210c04e1c3a?version=3.17.2

File: packages/taikoon-ui/src/components/sections/Collapsible.section.svelte [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+    content: string;
+  }[] = [];
+
+  const PlusIcon = Icons.PlusSign;
+  const XIcon = Icons.XSolid;
+</script>
+
+<Section height="fit">
+  <div
+    class={classNames(
```
Comment (KorbinianK) [inline packages/taikoon-ui/src/components/sections/Collapsible.section.svelte:20]: Dont use "inline" CSS, export them to variables.

Also the bridge uses daisyUI, which you also have as dependency. There is a collapse component included.

File: packages/taikoon-ui/src/lib/error/errors.ts [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+export class SendERC20Error extends Error {
+  name = 'SendERC20Error';
+}
+
+export class SendERC721Error extends Error {
+  name = 'SendERC721Error';
+}
+
+export class SendERC1155Error extends Error {
+  name = 'SendERC1155Error';
```
Comment (KorbinianK) [inline packages/taikoon-ui/src/lib/error/errors.ts:54]: Cleanup unused errors

File: packages/taikoon-ui/src/lib/ipfs/config.ts [hunk 5]
```diff
@@ -0,0 +1,3 @@
+//export const IPFS_GATEWAY = 'https://ipfs.4everland.io/ipfs/'
+//export const IPFS_GATEWAY = 'http://localhost:8080/ipfs/'
+export const IPFS_GATEWAY = 'https://taikoons-fake-ipfs-gateway.vercel.app/ipfs/';
```
Comment (KorbinianK) [inline packages/taikoon-ui/src/lib/ipfs/config.ts:3]: move to .env

**Review by KorbinianK — APPROVED** [review]

**Review by dantaik — APPROVED** [review]

File: packages/taikoon-ui/src/components/sections/Footer.section.svelte [hunk 6]
```diff
@@ -183,6 +186,8 @@
     'text-content-secondary',
     'font-normal',
   );
+
+  $: hoveredIcon = 'none' as ISocialLink;
 </script>
 
 <Section height={windowSize === 'sm' ? 'fit' : 'min'} background="footer" class={sectionClasses} width="xl">
```
Comment (bennettyong) [inline packages/taikoon-ui/src/components/sections/Footer.section.svelte:193]: lgtm but i think the footer section vh might be better suited at 55vh

**Review by bennettyong — APPROVED** [review]

File: packages/taikoon-ui/src/components/sections/Footer.section.svelte (hunk 6 above)
Comment (bearni95) [reply packages/taikoon-ui/src/components/sections/Footer.section.svelte:193]: Tackling on future PR

**Pull request merged**